app/Http/routes.php

```diff
@@ -52,7 +52,9 @@
     // Necessary because IDK and please let's fix this @todo
     Route::post('talks/{id}', 'TalksController@update');
     Route::post('conferences/{id}', 'ConferencesController@update');
+    Route::post('bios/{id}', 'BiosController@update');
```

Comment:
https://github.com/laravel/framework/issues/616

Basically the form builder doesn't infer the method when you use the action or route options (unfortunately). So can resolve easily by just adding a `'method' => 'put'` as part of the array when opening the form on the edit pages.